TreeMaker/python/makeJetVars.py

```diff
@@ -4,33 +4,25 @@
 def makeGoodJets(self, process, JetTag, suff, storeProperties, SkipTag=cms.VInputTag(), jetConeSize=0.4):
     from TreeMaker.TreeMaker.TMEras import TMeras
     from TreeMaker.Utils.goodjetsproducer_cfi import GoodJetsProducer
-    GoodJets = GoodJetsProducer.clone()
-    TMeras.TM2016.toModify(GoodJets,
+    GoodJets = GoodJetsProducer.clone(
         TagMode                   = cms.bool(True),
         JetTag                    = JetTag,
-        jetPtFilter               = cms.double(20 if self.fastsim else 30),
+        jetPtFilter               = cms.double(170 if jetConeSize==0.8 else 30),

```

Comment:
What happened to all of the fastsim parts? Does this not matter because you'll need all jet pt even for fastsim?

Reply:
Yes, in fact especially for fastsim, since that's one of the places we always evaluate JEC/JER uncertainty.